### DISCUSSION TITLE
RX MRF-Tax ID

### DISCUSSION THREAD

**Discussion by MMath19103** [CMSgov/price-transparency-guide#82] [Q&A] [answered]
Would like to confirm that the Tax ID in the RX Drug Price Object is the issuer or carrier's Tax ID or EIN
Thanks

**Comment by shaselton-usds** [comment, marked as answer]
Yes. You can look at the documentation found here: https://github.com/CMSgov/price-transparency-guide/blob/master/schemas/prescription-drugs/README.md#rx-file-schema

The `plan_id` is for the `plan_name` which is "The plan name and name of plan sponsor and/or insurance company"

**Reply by MMath19103** [reply]
Hi - thanks for the response Just to be sure we are talking about the same thing: 
In the drug price object the TIN is the plan tin?
.
.
.
historical_net_price	=Historical Net Price
historical_net_reporting_period	=Historical Net Price Reporting Period
negotiated_rate	=Negotiated Rate
administrative_fee	=Administrative Fee
dispensing_fee	=Dispensing Fee
transaction_fee	=Transaction Fee
**_tin	Tax Identification Number_**
service_code	=Place of Service Code
npi	=National Provider Identifier
pharmacies	=Providers
.
.
.
